Automattic/wp-calypso
# Update/onboarding medium importer style

#### Changes proposed in this Pull Request

This PR inherits logic behind wp-admin `ImportingPane` and `UploadingPane` to add additional styles we find in TAK5FpwTvfbENJDfHc7Aq1-fi.

* Add additional style changes to Medium import workflow
* New `ImportingPane` component
* New `UploadingPane` component
* WPCOM `WPCOM_REST_API_V2_Endpoint_Is_Site_Importable::is_site_importable` will returns additional data. Add a `favicon` and `title` string fields to `UrlData` type.

#### Testing instructions

* Go to: http://calypso.localhost:3000/start/importer/capture?siteSlug=YOUR_SITE
* Input a Medium file URL such as https://medium.com/@worthy_shadows_goose_107
* Load a medium `.zip` file
* See that the visual changes are the same as the TAK5FpwTvfbENJDfHc7Aq1-fi

#### Important
The author list component is missing the favicon because Medium `analyze-call` returns `null`. Try for example `https://medium.com/@worthy_shadows_goose_107`.

This can be added in a new PR.

Related to #59036.
Related to D72416-code.
